api/app/Console/Commands/CreateDigitalCareersPools.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\User;
+use App\Models\Pool;
+use App\Models\Skill;
+use App\Models\Classification;
+use Database\Helpers\ApiEnums;
+use Illuminate\Console\Command;
+use Carbon\Carbon;
+
+class CreateDigitalCareersPools extends Command
+{
+    // this command creates 32 pools, 4 classifications by 8 streams
```

Comment:
Maybe add a comment on when these commands can be safely deleted in case we forget later.